lib/index.js

```diff
@@ -238,7 +260,7 @@ module.exports.renderSync = function(options) {
 
   if (importer) {
     options.importer = function(file, prev) {
-      return { objectLiteral: importer(file, prev) };
+      return importer(file, prev);
```

Comment:
Note: `render` (the `async` variant) allows both callback as well as immediate return.